src/hash.cr

```diff
@@ -1476,14 +1476,18 @@ class Hash(K, V)
   # Removes a list of keys out of hash.
   #
   # ```
-  # h = {"a" => 1, "b" => 2, "c" => 3, "d" => 4}.reject!("a", "c")
-  # h # => {"b" => 2, "d" => 4}
+  # {"a" => 1, "b" => 2, "c" => 3, "d" => 4}.reject!(["a", "c"]) # => {"b" => 2, "d" => 4}
```

Comment:
This would conceal the fact that the operation is done in-place (in contrast to `#reject`), so `h` must remain, and both lines should have the same value:

```crystal
h = {...}
h.reject!(...) # => ...
h              # => ...
```

Ditto for below.

Reply:
that sounds good I can make this 2 lines.